pkg/agent/cniserver/pod_configuration.go

```diff
@@ -213,12 +215,12 @@ func (pc *podConfigurator) configureInterfaces(
 
 	// Check if the OVS configurations for the container exists or not. If yes, return immediately. This check is
 	// used on Windows, as kubelet on Windows will call CNI Add for infrastructure container for multiple times
-	// to query IP of Pod. But there should be only one OVS port created for the same Pod. And if the OVS port is added more than
-	// once, OVS will return an error.
+	// to query IP of Pod. But there should be only one OVS port created for the same Pod (identified by its sandbox
+	// container ID). And if the OVS port is added more than once, OVS will return an error.
 	// See https://github.com/kubernetes/kubernetes/issues/57253#issuecomment-358897721.
-	_, found := pc.ifaceStore.GetContainerInterface(podName, podNameSpace)
+	_, found := pc.ifaceStore.GetContainerInterface(containerID)
```

Comment:
Is it possible that this containerID is the workload container on Windows? For I didn't see the logics to find the infraContainerID before.

Reply:
No, createOVSPort is false for workload container, it doesn't try to create ovs port for workload container whether the port exists or not.